specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/examples/GetMaintenanceWindows.json

```diff
@@ -0,0 +1,28 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "Default-SQL-SouthEastAsia",
+    "workspaceName": "testsvr",
+    "sqlPoolName": "testdb",
+    "maintenanceWindowName": "current",
+    "api-version": "2019-06-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "properties": {
+          "timeRanges": [
```

Comment:
instead of modelling this as an array you could have one time range per maintenance window and allow creation of multiple maintenance windows (not just current), then list, get, put, delete are more intuitive. you could optionally also support a patch as well